### DISCUSSION TITLE
Unable to open Waters raw file in mzmine.

### DISCUSSION THREAD

**Discussion by vamskca** [mzmine/mzmine#1790] [Q&A]
Hello, I am new to metabolomics. I tried converting .raw files from waters to .mzml. I was able to make msconvertGUI work.
But for some reason one of my project files with just a single marker (function) in the raw file doesn't get converted to raw. Not sure why. So I tried mzmine. But mzmine is unable to import any of my waters .raw files. When I try to import, it shows the .raw files as empty folders and tries opening them instead of importing them.

**Comment by SteffenHeu** [comment]
Please follow this guide: https://mzmine.github.io/mzmine_documentation/data_conversion.html#waters
